wetmap/src/components/newModals/partnerAccountRequest/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PartnerAccountRequestView from './view';
+import { useContext } from 'react';
+import { Form } from './form';
+import { ModalHandleProps } from '../../reusables/modal/types';
+import { SessionContext } from '../../contexts/sessionContext';
+import { createPartnerAccountRequest } from '../../../supabaseCalls/partnerSupabaseCalls';
+
+type PartnerAccountRequestpProps = Partial<ModalHandleProps>;
+
+export default function PartnerAccountRequest(props: PartnerAccountRequestpProps) {
+  const { activeSession } = useContext(SessionContext);
+
+  const onSubmit = (data: Form) => {
+    createPartnerAccountRequest({
+      ...data,
+      UserId: activeSession?.user?.id || null,
```

Comment:
for form submissions we have been using the profile context rather than the session one. Also we need a UserId for a successful submission so we can tie the request back to the user who requested it and upgrade their account accordingly